requirements/requirements-test.txt

```diff
@@ -20,3 +20,4 @@ pydantic
 ray
 peft>=0.7.1
 #auto-gptq now not support torch1.12
+galore_torch
```

Comment:
This is written in `requirements.txt` and can be removed here.

Reply:
updated.